src/lib/utils/map.ts

```
@@ -41,8 +42,37 @@ export const getFeatureType = (
-    if (featureTypes.length) return featureTypes[0]
+// todo: invullen
+// @param {FeatureType[]} featureType - Array of possible feature types
+// @param {GeoJsonProperties} properties - Properties of the GeoJSON feature
+// @returns {FeatureType | null} - Matching feature type or null
+export const getFeatureIconUrl = (
+  featureType: FeatureType[],
+  properties: GeoJsonProperties
+): string | null => {
+  if (properties) {
+    const featureTypes = featureType.filter((feature: FeatureType) =>
+      properties.hasOwnProperty(feature.idField)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)